iop/fs/bdm/src/part_driver.c

```diff
@@ -129,7 +150,6 @@ int part_connect(struct block_device* bd)
     M_DEBUG("%s\n", __func__);
 
     // Filter for supported partition IDs:
-    // - First sector of disk can be MBR partition
     // - 0x05 = extended MBR partition with CHS addressing
```

Comment:
Is this not true?

Reply:
Yes, this also was for Removable vs MBR, if there is fallback to REmovabvle this is not needed